src/data-access/users.ts

```diff
@@ -41,11 +42,13 @@ export async function checkUserExistence(payId: string): Promise<boolean> {
 export async function insertUser(
   payId: string,
   addresses: readonly AddressInformation[],
+  identityKey: string | undefined = undefined,
```

Comment:
I think you should be able to make this an optional parameter.

```suggestion
  identityKey?: string,
```

Reply:
Lol. Yes. You are right, fixed this.